src/components/events/SkillEndorsementPanel.tsx

```
@@ -0,0 +1,442 @@
+  const queryClient = useQueryClient();
+
+  const [dismissed, setDismissed] = useState(false);
+  const [selectedVolunteer, setSelectedVolunteer] = useState<string>("");
+  const [skillTag, setSkillTag] = useState<string>("");
+  const [comment, setComment] = useState<string>("");
+  const [viewerWeight, setViewerWeight] = useState<number>(0.6);
+
+  // The event itself — endorsements unlock only once it has ended.
+  const {
+    data: event,
+  } = useQuery<{
```

ESLint

[error] Replace `⏎····data:·event,⏎·` with `·data:·event`

(prettier/prettier)